app/models/remote_execution_feature.rb

```diff
@@ -1,7 +1,7 @@
 class RemoteExecutionFeature < ActiveRecord::Base
   attr_accessible :label, :name, :provided_input_names, :description, :job_template_id
 
-  validate :label, :name, :presence => true, :unique => true
+  validates :label, :name, :presence => true, :uniqueness => true
```

Comment:
no, I think the database_cleaner is still valuable patch, just remove the `validates` change I'll use the ticket number it that combined PR